test/jdk/java/awt/color/ICC_ColorSpace/SimpleSRGBToFromCIEXYZ.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
The test is fine but since as you note, it is based on a test from JBS, written by an (at the time) Sun employee, I'd expect (c) to see Oracle here .. or at least joint (c).

Reply:
Sure. the test is updated.